src/svelte/form/TextArea.svelte

```diff
@@ -7,6 +7,7 @@
   export let value;
   export let name: string;
   export let label: string;
+  export let countCharactersLeftLabel: string;
```

Comment:
We still allows `undefined` as a legal value here, since we provide a default value in  `countCharacters`, right?

```suggestion
  export let countCharactersLeftLabel: string | undefined;
```